CMakeLists.txt

```diff
@@ -100,6 +100,7 @@ add_subdirectory(src/IPGenerate)
 add_subdirectory(src/TextEditor)
 add_subdirectory(src/Utils)
 add_subdirectory(src/IpConfigurator)
+add_subdirectory(src/PinAssignmentView)
```

Comment:
Change to PinAssignment to align with other srcs.

Reply:
Done